src/runtime_src/core/pcie/driver/linux/xocl/subdev/ert_user.c

```diff
@@ -17,13 +17,35 @@
 
 #include "../xocl_drv.h"
 
-#define	ERT_MAX_SLOTS 	128
+#define	ERT_MAX_SLOTS		128
 
-#define	ERT_STATE_GOOD	0x1
-#define	ERT_STATE_BAD	0x2
+#define	ERT_STATE_GOOD		0x1
+#define	ERT_STATE_BAD		0x2
 
 
-//#define	SCHED_VERBOSE	1
+//#define	SCHED_VERBOSE		1
+
+
+
+/* ERT gpio config has two channels 
+ * CHANNEL 0 is control channel :
+ * BIT 0: 0x0 Selects interrupts from embedded scheduler HW block
+ * 	  0x1 Selects interrupts from the CU INTCs
+ * BIT 2-1: TBD
+ *
+ * CHANNEL 1 is status channel :
+ * BIT 0: check microblazer status
+ */
+
+#define GPIO_CFG_CHANNEL_0	0x0
```

Comment:
can we name channel 0 as GPIO_CONFIG_CHANNEL and channel 1 as GPIO_STATUS_CHANNEL?